### PR TITLE
Convert YarnHelper to module

### PR DESCRIPTION
**Pull request by severisv**
#1721 #1733

### CODE REVIEW

File: src/app/Fake.JavaScript.Yarn/Yarn.fs [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+    let private yarn setParams = defaultYarnParams |> setParams |> run
+
+
+    /// Run `yarn install`
+    /// ## Parameters
+    /// - 'setParams' - set command parameters
+    /// ## Sample
+    ///
+    ///        Target.Create "Restore-frontend" (fun _ ->
```
Comment (matthid) [inline src/app/Fake.JavaScript.Yarn/Yarn.fs]: maybe adapt docs (`Target.create`). Or we leave target out completely?

File: src/legacy/FakeLib/YarnHelper.fs [hunk 2]
```diff
@@ -29,6 +29,7 @@ let private yarnFileName =
               proc.StandardOutput.ReadLine()
             | _ -> "/usr/bin/yarn"
 
+[<System.Obsolete("FAKE0001 Use the Fake.JavaScript.Yarn module instead")>]
```
Comment (matthid) [inline src/legacy/FakeLib/YarnHelper.fs]: We should add the attribute to the module as well.

File: src/app/Fake.JavaScript.Yarn/Yarn.fs [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+    /// The list of supported Yarn commands. The `Custom` alternative
+    /// can be used for other commands not in the list until they are
+    /// implemented
+    type YarnCommand =
+    | Install of InstallArgs
+    | Custom of string
+
+    /// The Yarn parameter type
+    [<CLIMutable>]
```
Comment (matthid) [inline src/app/Fake.JavaScript.Yarn/Yarn.fs]: no more climutable

**Comment by severisv** [comment]
I have updated the branch fixing the requested changes, let me know if there's anything else. 
I'll submit an update removing the bloat from the npm module aswell

**Comment by matthid** [comment]
Thanks!

**Pull request merged**